community/blog/2022-05 The long overdue Arcus EventGrid v3.2 release/README.md

```diff
@@ -0,0 +1,155 @@
+# The long overdue Arcus EventGrid v3.2 release
+Version v3.2 of the Arcus EventGrid library was released recently, almost two years after v3.1. Here's a long overdue quick run-down of the new features and changes in this release.
+
+## EventGrid authorization with Arcus secret store
+We provide support for endpoint validation, when implementing your own custom web hook. This validation allows to secure your web hook with a secret key (taken from the query string or an HTTP header).  
+This is needed, because Azure EventGrid is send a validation request to a newly configured web hook, in order to prevent people leveraging Azure EventGrid to bring down a 3rd party API. 
```

Comment:
```suggestion
This is needed, because Azure EventGrid sends a validation request to a newly configured web hook, in order to prevent people leveraging Azure EventGrid to bring down a 3rd party API. 
```

Reply:
Ah, thx 😄